### PR TITLE
Feature/wish list

### PR DESCRIPTION
**Pull request by CowboyLynk**
#2 
Wishlist is now completely functional. Meaning you can add and delete courses.

### CODE REVIEW

File: courses_database/static/js/utils.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                button_text = "Remove Course"
+            }
+            $("#displayed_button").html(button_text)
+        } else {
+            $("#displayed_content").hide();
+        }
+    }
+
+    addToWishList(){
+        addToWishList(this.subject, this.course_id, this.title)
```
Comment (DanielMcCrystal) [inline courses_database/static/js/utils.js]: This naming is confusing. Where is the other addToWishList?

**Comment by DanielMcCrystal** [comment]
I'll go ahead and approve this since the feature is from the original version, but in general I think we need to add more documentation to the javascript. It doesn't have to be line by line, but at least have comments above functions that describe their purpose, and more importantly, the context in which they might be called.

File: courses_database/static/js/utils.js (hunk 1 above)
Comment (CowboyLynk) [reply courses_database/static/js/utils.js]: Ah okay yeah, that's super confusing. That's calling the global addToWishList in main.js. Any suggestions to fix this?

**Comment by CowboyLynk** [comment]
Documenting all the Javascript is an open issue (#20). I was waiting for these two pull requests to be approved though. That's my next task.

File: courses_database/static/js/utils.js (hunk 1 above)
Comment (CowboyLynk) [reply courses_database/static/js/utils.js]: Maybe renaming it to 'globalAddToWishList'?

**Pull request merged**